webrender/src/frame_builder.rs

```diff
@@ -365,13 +365,6 @@ impl FrameBuilder {
                 // must be drawn with a transparent background, unless the parent stacking context
                 // is the root of the page
                 let isolation = &mut self.stacking_context_store[parent_index.0].isolation;
-                if *isolation != ContextIsolation::None {
```

Comment:
what if it's `ContextIsolation::Items`? overwriting it with `Full` would screw up the follow up logic

Reply:
Thank you for the feedback.

Maybe we could relax the condition here instead of removing it and make the stacking context able to contain multiple child stacking context with blend mode?

According to the code, we only use ContextIsolation::Items for elements which contain preserve-3d transform-style.
And in gecko, an additional stacking context will be created to contain stacking contexts with blend mode.
With this mechanism, we can prevent isolation conflict in a stacking context.
I am not sure if Servo has this mechanism, but I think it might be reasonable to do that.

I will push another pr for this.

Reply:
There is a PR nearly ready to merge which should solve most (or all?) of the isolation issues - https://github.com/servo/webrender/pull/2031. Ideally, we'd get that to land, and then perhaps you could investigate if there are any remaining isolation failures and fix them on top of that patch (since it changes the way we handle stacking contexts internally very significantly)?

Reply:
Oh, thank you for the feedback! Then I will cancel the pr and wait for the issue solved.

Reply:
Sounds good, thank you!